translations/Readme.md

```diff
@@ -0,0 +1,37 @@
+# About Translation
+
+## How to use the translation utilities
+
+### On Windows
+
+#### 1. Adding translation `(.ts)` files
+
+1. Open `lupdate.cmd` in editor. (e.g, VSCode or Notepad, etc.)
+2. Append your new file name to the end of the first line, in the format **locale_**_language-code_**.ts** (e.g, locale_de, local_zh-cn, etc.).
+3. Run the command file in cmd.
+4. You will find the new file in the same location as the `lupdate.cmd`.
+
+#### 2. Generating / Updating `.qm` files
+
+Just double click on `lrelease.cmd` or run it in `cmd.exe`.
+
+### On Linux / macOS
+
+Use the `translation_utility.sh`.
+
+This is a 2-in-1 utility, which enables you to process `.ts` and `.qm` files in an efficient way.
+
+Before running the script, you have to assign permissions to `translation_utility.sh` using `sudo chmod u+x`.
+
+Then you can double click on it or run it in a shell. The instructions are shown in the UI.
```

Comment:
`Between updating/generating the .ts files and the .qm files, translate the strings, done the same way as in step 3 for Windows, see above`

Reply:
I don't got the meaning

Reply:
After creating or updating the .ts file and before creating/updating the .qm files, you'd need to to the real actualy work of translating the strings